docarray/array/array/io.py

```diff
@@ -16,20 +16,21 @@
     Dict,
     Generator,
     Iterable,
+    List,
     Optional,
-    Sequence,
     Tuple,
     Type,
     TypeVar,
     Union,
 )
 
+import pandas as pd
```

Comment:
this should be an optional dependency, therefore we cannot import from it directly